### ISSUE TITLE
Migrating from TF1 to TF2, loss not going down using F-RCNN Resnet50

### ISSUE THREAD

**Issue by Mustavi** [tensorflow/tensorflow#45192]
So I've been trying to migrate from Tensorflow Version 1.14 to 2.3.0. My work usually involves performing transfer learning on Faster R-CNN Resnet 50 Coco Trained Models for object detection. When I used to train on Tensorflow 1.14 using the adam optimizer, I would see the loss go down after 30k steps of training. However, on TF2, the loss seems to be more of a straight line. 

It would help me greatly if someone could help me figure out why the loss is not going down. Thanks in advance. :)

TF2 Tensorboard Total Loss on [faster_rcnn_resnet50_v1_1024x1024_coco17_tpu-8](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md)
![image](https://user-images.githubusercontent.com/32026314/100305789-6e000780-2fd4-11eb-8c23-2599a735b480.png)

TF1 Tensorboard Total Loss on [faster_rcnn_resnet50_coco_2018_01_28](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf1_detection_zoo.md)
![Screenshot from 2020-11-23 10-49-57](https://user-images.githubusercontent.com/32026314/100306629-6c374380-2fd6-11eb-9279-a75a8f01c5b5.png)

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): NO
- OS Platform and Distribution: Ubuntu 20.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: NO
- TensorFlow installed from (source or binary): PIP3
- TensorFlow version (use command below): 2.3.0
- Python version: 3.8
- CUDA/cuDNN version: 10.1
- GPU model and memory: 980 Ti, 6GB

**Comment by Mustavi** [comment]
My Pipleline Config


model {
  faster_rcnn {
    num_classes: 1
    image_resizer {
      fixed_shape_resizer {
        height: 100
        width: 100
      }
    }
    feature_extractor {
      type: 'faster_rcnn_resnet50_keras'
      batch_norm_trainable: true
    }
    first_stage_anchor_generator {
      grid_anchor_generator {
        scales: [0.25, 0.5, 1.0, 2.0]
        aspect_ratios: [0.5, 1.0, 2.0]
        height_stride: 16
        width_stride: 16
      }
    }
    first_stage_box_predictor_conv_hyperparams {
      op: CONV
      regularizer {
        l2_regularizer {
          weight: 0.0
        }
      }
      initializer {
        truncated_normal_initializer {
          stddev: 0.01
        }
      }
    }
    first_stage_nms_score_threshold: 0.0
    first_stage_nms_iou_threshold: 0.7
    first_stage_max_proposals: 300
    first_stage_localization_loss_weight: 2.0
    first_stage_objectness_loss_weight: 1.0
    initial_crop_size: 14
    maxpool_kernel_size: 2
    maxpool_stride: 2
    second_stage_box_predictor {
      mask_rcnn_box_predictor {
        use_dropout: false
        dropout_keep_probability: 1.0
        fc_hyperparams {
          op: FC
          regularizer {
            l2_regularizer {
              weight: 0.0
            }
          }
          initializer {
            variance_scaling_initializer {
              factor: 1.0
              uniform: true
              mode: FAN_AVG
            }
          }
        }
        share_box_across_classes: true
      }
    }
    second_stage_post_processing {
      batch_non_max_suppression {
        score_threshold: 0.0
        iou_threshold: 0.6
        max_detections_per_class: 100
        max_total_detections: 300
      }
      score_converter: SOFTMAX
    }
    second_stage_localization_loss_weight: 2.0
    second_stage_classification_loss_weight: 1.0
    use_static_shapes: true
    use_matmul_crop_and_resize: true
    clip_anchors_to_image: true
    use_static_balanced_label_sampler: true
    use_matmul_gather_in_matcher: true
  }
}

train_config: {
  batch_size: 1
  sync_replicas: true
  startup_delay_steps: 0
  replicas_to_aggregate: 8
  num_steps: 150000
  optimizer {
   adam_optimizer {
      learning_rate {
        manual_step_learning_rate {
          initial_learning_rate: 0.0002
          schedule {
            step: 50000
            learning_rate: 0.0002
          }
          schedule {
            step: 100000
            learning_rate: 0.0002
          }
        }
      }
    }
    use_moving_average: false
  }
  fine_tune_checkpoint_version: V2
  fine_tune_checkpoint: ""
  fine_tune_checkpoint_type: "detection"
  data_augmentation_options {
    random_horizontal_flip {
    }
  }

  data_augmentation_options {
    random_adjust_hue {
    }
  }

  data_augmentation_options {
    random_adjust_contrast {
    }
  }

  data_augmentation_options {
    random_adjust_saturation {
    }
  }


  max_number_of_boxes: 100
  unpad_groundtruth_tensors: false
  use_bfloat16: true  # works only on TPUs
}
train_input_reader {
  label_map_path: ""
  tf_record_input_reader {
    input_path: ""
  }
}
eval_config {
  metrics_set: "coco_detection_metrics"
  use_moving_averages: false
  batch_size: 1
}
eval_input_reader {
  label_map_path: ""
  shuffle: false
  num_epochs: 1
  tf_record_input_reader {
    input_path: ""
  }
}

**Comment by ravikyram** [comment]
@Mustavi 

This issue is more suitable for TensorFlow Models repo. Please post it on Tensorflow Models repo from [here.](https://github.com/tensorflow/models/issues/new/choose) Thanks!
